src/wp-includes/block-supports/layout.php

```diff
@@ -883,17 +883,63 @@ function wp_restore_group_inner_container( $block_content, $block ) {
 		return $block_content;
 	}
 
-	$replace_regex   = sprintf(
+	/**
+	 * This filter runs after the layout classnames have been added to the block, so they
+	 * have to be removed from the outer wrapper and then added to the inner.
+	*/
```

Comment:
```suggestion
	/*
	 * This filter runs after the layout classnames have been added to the block, so they
	 * have to be removed from the outer wrapper and then added to the inner.
	 */
```
Improve inline comments format per [PHP Documentation Standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-inline-comments)